model/lifecycle.go

```diff
@@ -135,11 +171,6 @@ func setTaskActivationForBuilds(buildIds []string, active, withDependencies bool
 			return errors.Wrap(err, "deactivating tasks")
 		}
 	}
-
-	if err := UpdateVersionAndPatchStatusForBuilds(buildIds); err != nil {
```

Comment:
Can you explain why we remove this? Won't this break places that use this?

Reply:
i meant to copy it and i think i cut it sorry